src/main.rs

```diff
@@ -41,7 +42,7 @@ Usage:
     hemtt init
     hemtt create
     hemtt addon <name>
-    hemtt build [<addons>] [--release] [--force] [--nowarn] [--opts=<addons>] [--skip=<addons>]
+    hemtt build [<addons>] [--release] [--force] [--nowarn] [--opts=<addons>] [--skip=<addons>] [---jobs=<n>]
```

Comment:
```suggestion
    hemtt build [<addons>] [--release] [--force] [--nowarn] [--opts=<addons>] [--skip=<addons>] [--jobs=<n>]
```

Reply:
@synixebrett should we start putting short flags where they exist in this line to make it shorter? They are explained below and most of them are the same as any CLI tool.

Reply:
Probably should yeah, it's only going to get longer.